Sources/TensorFlow/Core/Runtime.swift

```diff
@@ -39,6 +39,17 @@ import Glibc
 
 import CTensorFlow
 
+#if os(Windows)
+// NOTE(abdulras) although the function is racy, we do not really care as the
```

Comment:
Minor comment: is this note something that you plan to take action on sometime? If not, I'd recommend removing the `abdulras` reference (especially since it's a `NOTE` and not a `TODO`).
```suggestion
// NOTE: although the function is racy, we do not really care as the
```

Reply:
I don't think that there is anything that is actionable there (at least for the short term).  I think that moving away from the environment variables and into proper context to the function calls is better, but that would be a longer term thing and need to work out the APIs with the other side.